code-challenges.test.js

```diff
@@ -10,18 +18,68 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+// ===================================== CHALLENGE #1 =================================================
+
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
+// const fibLength1 = 6;
 // Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
+// const fibLength2 = 10;
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// const fibLength3 = 1;
+// Expected output: "1 in NOT greater than 2"
+
+describe("fibSequenceNum", () => {
+  it("The function takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    const fibLength1 = 6;
+    const fibLength2 = 10;
+    const fibLength3 = 1;
+    expect(fibSequenceNum(fibLength1)).toEqual([1, 1, 2, 3, 5, 8]);
+    expect(fibSequenceNum(fibLength2)).toEqual([
+      1, 1, 2, 3, 5, 8, 13, 21, 34, 55,
+    ]);
+    expect(fibSequenceNum(fibLength3)).toEqual("1 in NOT greater than 2");
+  });
+});
+
+//ReferenceError: fibSequenceNum is not defined
+
 // b) Create the function that makes the test pass.
 
+// ============== <<<<< PSEUDO CODE >>>>> ===========================
+// the function will take in a number that will essentially set the fibonacci array's length
+//we need to create fibonacci array using simple formula or we can just copy an array from internet,
```

Comment:
Not entirely sure what you are meaning here, but would strongly recommend not copying from the internet.